### DISCUSSION TITLE
Export Samsung Pass valut to csv

### DISCUSSION THREAD

**Discussion by 99vitts99** [roddhjav/pass-import#202] [Ideas]
Like many samsung users, I discovered when needed that the hundreds of passwords saved in samsung pass cannot be exported in csv format.
In particular, the data is exported in a file with the .spass extension which is encrypted at the time of export with a password of your choice.
A method that allows, knowing the export password, to convert the .spass file into a txt or csv file (in order to be exported to another password manager) would be very useful

**Comment by roddhjav** [comment]
To add this support, I would need to have a .spass file with the [reference vault](https://github.com/roddhjav/pass-import/blob/master/tests/assets/references/main.yml) and [master password](https://github.com/roddhjav/pass-import/blob/71e2e65c67c5cb435c9d74995ba633f4d1fd0069/tests/__init__.py#L195) . If you can provide me with such a file (and some doc) that could be done.

**Reply by 99vitts99** [reply]
Thank you for responding me. 
I have created a reference valut with the only one following element:

site: hello.com
username: hello
password: hello

Then i have exported the valut to an .spass file* using this master password:

Master password: Hellohello0?


*the .spass file is available here: https://mega.nz/file/xRA1gAyb#uGuO6c_SNiIIiN5NY2YNn2_j2ck5LsljoYn5PThvkks

Thanks

**Reply by roddhjav** [reply]
I will need a vault with the exact content of the reference vault as it has been crafted in such a way to detect weird edge cases that your simple vault will not cover. Furthermore, the test suite is expecting this specific vault to work.

**Reply by 99vitts99** [reply]
I'm sorry but I'm afraid I don't understand. The valut I provided contains only the item I listed above. If by "exact content" you mean the content of the spass file before encryption (ie how the data is sorted within the .spass file by samsung pass before encrypting it) this I'm afraid is not possible to know.

The only thing I can do is provide several reference values all encrypted with the same password but with different login elements to see if there is any similarity
